### PR TITLE
Describe new changelog and access log request/response properties

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

**Review by venelinkotsev — APPROVED** [review]

**Review by vulkodava — APPROVED** [review]

File: version1/04 - Web services.md [hunk 1]
```diff
@@ -1150,10 +1152,18 @@ Content-Type: application/json
 ###### Details
 
 - **type**
-  - object type \[Saksmappe, Mappe, Moetemappe, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
-    AbstraktRegistrering, Dokument\]
+  - object type \[Arkiv, Arkivskaper, Arkivdel, Klassifikasjonssystem, Klasse, Saksmappe, Mappe, Moetemappe, Moetedeltaker, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
```
Comment (dkumanov) [inline version1/04 - Web services.md:1156]: Shouldn't we include `Sakspart` and `Korrespondansepart`?

Comment (ivaylomitrev) [reply version1/04 - Web services.md:1156]: I think I've missed them, yes. I'll add a suggestion on the next line and commit it.

File: version1/04 - Web services.md [hunk 2]
```diff
@@ -1150,10 +1152,18 @@ Content-Type: application/json
...
+    AbstraktRegistrering, Dokument, Presedens\]
```
Comment (ivaylomitrev) [inline version1/04 - Web services.md]:
```suggestion
    AbstraktRegistrering, Dokument, Presedens, Korrespondansepart, Sakspart\]
```

File: version1/04 - Web services.md (hunk 1 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md:1156]: Added with https://github.com/documaster/noark5-web-services/pull/70/commits/4d7fcc4d19830942c8f2878f4da0225376ee6f27

File: version1/04 - Web services.md [hunk 3]
```diff
@@ -1150,10 +1152,18 @@ Content-Type: application/json
...
-    AbstraktRegistrering, Dokument\]
+  - object type \[Arkiv, Arkivskaper, Arkivdel, Klassifikasjonssystem, Klasse, Saksmappe, Mappe, Moetemappe, Moetedeltaker, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
+    AbstraktRegistrering, Dokument, Presedens, Korrespondansepart, Sakspart\]
+  - optional, but recommended
 - **id**
-  - the ID of the object
+  - the ID or system ID of the object
+  - required
+- **includeChildChanges**
+  - if true, marker events for creates/updates/deletes will be returned for supported entities
```
Comment (dkumanov) [inline version1/04 - Web services.md]:
```suggestion
  - if true, marker events for creates/moves/deletes will be returned for supported entities
```

File: version1/04 - Web services.md [hunk 4]
```diff
@@ -1309,6 +1326,8 @@ Content-type: application/json
   - the date of creating the object
 - **userName**
   - the username accessing the object
+- **userId**
```
Comment (dkumanov) [inline version1/04 - Web services.md:1330]: The `userId` should also be added to the download log response.

File: version1/04 - Web services.md (hunk 3 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Technically, child _updates_ could be included as well. For example, correspondence party name change when fetching a registry entry's changelog.

File: version1/04 - Web services.md (hunk 4 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md:1330]: Added w/ https://github.com/documaster/noark5-web-services/pull/70/commits/5da358a44274f719cb30ba9e5c18a8e018e8f5fa

File: version1/04 - Web services.md [hunk 5]
```diff
@@ -1267,13 +1283,13 @@ Content-Type: application/json
...
+- **objectType**
   - the object type to filter by \[Saksmappe, Mappe, Moetemappe, Journalpost, Basisregistrering, Moeteregistrering,
     Arkivnotat, Dokument, Dokumentversjon\]
-  - must be provided if objectUUIDs is
-- **objectUUIDs** (conditionally optional)
+  - optional, but recommended when objectUUIDs is specified
+- **objectUUIDs**
   - the list of object system ids to filter by
-  - must be provided if objectType is
+  - optional
```
Comment (dkumanov) [inline version1/04 - Web services.md:1293]: I don't think the `objectUUIDs` is optional.

File: version1/04 - Web services.md (hunk 3 above)
Comment (dkumanov) [reply version1/04 - Web services.md]: But aren't these kind of changes reserved for the `nestChildChanges` flag?

**Comment by dkumanov** [comment]
No more comments from my side :cowboy_hat_face:

File: version1/04 - Web services.md (hunk 3 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: They are 😅 . Applying your suggestion

File: version1/04 - Web services.md (hunk 5 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md:1293]: For access logs, UUIDs are optional. We only require an ID for the changelog.

**Comment by ivaylomitrev** [comment]
I have responded to all comments now.

File: version1/04 - Web services.md (hunk 5 above)
Comment (dkumanov) [reply version1/04 - Web services.md:1293]: Hmm, yes, my bad :sweat_smile:

**Review by dkumanov — APPROVED** [review]

**Pull request merged**